addons/modules/fnc_getStringPos.sqf

```diff
@@ -0,0 +1,39 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getStringPos
+
+Description:
+    A function used to get the position of a string with a passed typeName.
```

Comment:
This description is a bit cryptic. A string does not have a position. Improve the short one-line description. You can add a second paragraph with more text that explains the function better, if needed.

(Perhaps the function should be named something having to do with parsing a string input... `getStringPos` feels...strange)